paramnormal/dist.py

```diff
@@ -904,6 +904,39 @@ def fit(cls, data, **guesses):
         return cls.param_template(R=b*sigma, loc=loc, sigma=sigma)
 
 
+class truncated_normal(BaseDist_Mixin):
+    dist = stats.truncnorm
+    param_template = namedtuple('params', ['lower', 'upper', 'mu', 'sigma'])
+    name = 'truncated normal'
+
+    @staticmethod
+    @utils.greco_deco
+    def _process_args(lower=None, upper=None, mu=None, sigma=None, fit=False):
+        a = None
+        b = None
+        if lower is not None and mu is not None and sigma is not None:
+            a = (lower - mu) / sigma
+
+        if upper is not None and mu is not None and sigma is not None:
```

Comment:
@ev-br sorry to bother you. any thoughts on how to deal with missing parameters when fitting? this method seems almost useless if you need 3 parameters just to fix one.

In other words, `scipy.stats.truncnorm.fit(fa=<guess>)` will work, but `paramnormal.truncated_normal.fit(lower=<guess>)`  won't since you also need `mu` and `sigma` to estimate `a`.

On the otherhand, if you're fitting `truncnorm` distributions and you have a guess for `a`, does it follow that you implicitly have a guess for sigma and mu?